### PR TITLE
Acceptance keep floc 1935

### PR DESCRIPTION
**Pull request by adamtheturtle**

### CODE REVIEW

File: admin/acceptance.py [hunk 1]
```diff
@@ -449,11 +449,14 @@ def main(reactor, args, base_path, top_level):
             make_dispatcher(reactor),
             configure_cluster(control_node=nodes[0], agent_nodes=nodes,
                               certificates=certificates))
+
+        control_node = nodes[0]
```
Comment (exarkun) [inline admin/acceptance.py]: How do we know the control service is running on this node and not another one?

Comment (exarkun) [inline admin/acceptance.py]: Or is this prescriptive rather than descriptive?

Comment (adamtheturtle) [inline admin/acceptance.py]: Just above this:

```python
configure_cluster(control_node=nodes[0], agent_nodes=nodes, ...
```

I think that this variable should be moved above that and used there too.

Comment (exarkun) [inline admin/acceptance.py]: Ah.  `nodes[0]` is passed to `configure_cluster` two lines above.  Please move this assignment up and use `control_nodes` as the value to pass to `configure_cluster` to make it more clear we're setting it up this way.

Comment (exarkun) [inline admin/acceptance.py]: :grin:

**Comment by exarkun** [comment]
Thanks.  Address that comment and then merge.

**Pull request merged**